added astrocloud init (#2383)
fishtown-analytics/docs.getdbt.com: website/docs/guides/orchestration/airflow-and-dbt-cloud/2-setting-up-airflow-and-dbt-cloud.md

> ## 4. Start the Docker container

Reviewer: ```suggestion
## 4. Start the Docker container

You can initialize an Astronomer project in an empty local directory using a Docker container, and then run your project locally using the `start` command.
```

> 1. Run the following commands to turn your directory into an Astronomer project and spin up the Docker container and start your local Airflow deployment:

Reviewer: ```suggestion
1. Run the following commands to initialize your project and start your local Airflow deployment:
```
